lib/filterx/filterx-scope.c

```diff
@@ -248,7 +248,7 @@ filterx_scope_register_variable(FilterXScope *self,
 }
 
 gboolean
-filterx_scope_foreach_variable(FilterXScope *self, FilterXScopeForeachFunc func, gpointer user_data)
+filterx_scope_foreach_variable_readonly(FilterXScope *self, FilterXScopeForeachFunc func, gpointer user_data)
```

Comment:
I'm fine with the rename, however for each style functions often have the restriction that the object can't be modified, so I don't understand why this is special.

Reply:
`foreach()` functions usually allow storing or modifying the returned objects. Since these come from multiple scopes, neither is allowed.